app/models/fact_name.rb

```diff
@@ -6,7 +6,7 @@ class FactName < ActiveRecord::Base
   validates_lengths_from_database
   has_many :fact_values, :dependent => :destroy
   has_many_hosts :through => :fact_values
-  attr_accessible :parent, :name
+  attr_accessible :parent, :name, :compose
```

Comment:
While you're here, both foreman_salt and chef appear to use :parent_id rather than :parent.  Would you mind adding it?

https://github.com/theforeman/foreman_chef/blob/master/app/models/foreman_chef/fact_importer.rb#L41
https://github.com/theforeman/foreman_salt/blob/master/app/services/foreman_salt/fact_importer.rb#L34